internal/show/versions.go

```diff
@@ -19,24 +19,44 @@ limitations under the License.
 package show
 
 import (
+	"bufio"
 	"context"
+	"fmt"
+	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/submariner-io/admiral/pkg/names"
 	"github.com/submariner-io/admiral/pkg/reporter"
 	"github.com/submariner-io/subctl/internal/constants"
+	"github.com/submariner-io/subctl/internal/pods"
 	"github.com/submariner-io/subctl/internal/show/table"
 	"github.com/submariner-io/subctl/pkg/cluster"
 	"github.com/submariner-io/submariner-operator/pkg/images"
+	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
 )
 
+const maxLogLinesToScan = 20
+
+var componentCmd = map[string]string{
+	names.NetworkPluginSyncerComponent: "submariner-networkplugin-syncer",
+	names.RouteAgentComponent:          "submariner-route-agent",
+	names.GatewayComponent:             "submariner-gateway",
+	names.GlobalnetComponent:           "submariner-globalnet",
+	names.ServiceDiscoveryComponent:    "lighthouse-agent",
+	names.LighthouseCoreDNSComponent:   "lighthouse-coredns",
+	names.OperatorComponent:            "submariner-operator",
+	names.MetricsProxyComponent:        "cat",
```

Comment:
This shouldn’t block this PR, but since we have a specific command for the metrics proxy now, we could add `--version` support to that.